### ISSUE TITLE
can rs/webrtc be used in native client side? or just for server use?

### ISSUE THREAD

**Issue by shushenghong** [webrtc-rs/webrtc#483]
if rs/webrtc can be a substude for google libwebrtc for native client use, can anyone help me how to build rs/webrtc for native client use? like ios objectc + rs/webrtc ; android java jni + rs/webrtc; windows c#/c++ + rs/webrtc?

**Comment by santoshakil** [comment]
I also need to know this.

**Comment by rainliu** [comment]
yes, webrtc-rs can be used for native client, but you need to take care of Audio/Video Capture, Encoding, Decoding and Rendering.

**Comment by santoshakil** [comment]
> yes, webrtc-rs can be used for native client, but you need to take care of Audio/Video Capture, Encoding, Decoding and Rendering.

Any plan in the near future to support "Audio/Video Capture, Encoding, Decoding, and Rendering"?

**Comment by shushenghong** [comment]
> yes, webrtc-rs can be used for native client, but you need to take care of Audio/Video Capture, Encoding, Decoding and Rendering.
I will write Audio/Video Capture, Encoding, Decoding and Rendering in platform native language.
I just need rs/webrtc to transport audio/video data, any document on this?

**Comment by overheat** [comment]
> > yes, webrtc-rs can be used for native client, but you need to take care of Audio/Video Capture, Encoding, Decoding and Rendering.
> 
> I will write Audio/Video Capture, Encoding, Decoding and Rendering in platform native language. I just need rs/webrtc to transport audio/video data, any document on this?

+1
I have the same requirement.
